modules/carto/src/api/maps-v3-client.js

```diff
@@ -124,6 +125,21 @@ function getUrlFromMetadata(metadata, format) {
 }
 
 export async function getData({type, source, connection, credentials, geoColumn, columns, format}) {
+  // Internally we split data fetching into two parts to allow us to
```

Comment:
I think we should trust the standard browser cache system to fetch the actual. In any case, I stille like the idea of split getData method 👍